### PR TITLE
admission/webhook: make test cert works with go1.9

### PR DESCRIPTION
**Pull request by dqminh**
The old certificate defines SAN for IP, but doesnt include DNS name.
Since Go 1.9 (https://go-review.googlesource.com/c/go/+/36696), this
will ignore the CN, makes the test fail with

```
http: TLS handshake error from 127.0.0.1:50686: remote error: tls: bad certificate
18419 admission.go:188] failed calling admission webhook "allow": Post https://127.0.0.1:40537/allow: x509: certificate is not valid for any names, but wanted to match webhook-test.default.svc
```

Regenerate the certificate with SAN removed to make test pass. We dont
seem to need it anyway.

cc @caesarxuchao

### CODE REVIEW

File: plugin/pkg/admission/webhook/gencerts.sh [hunk 1]
```diff
@@ -44,11 +41,7 @@ distinguished_name = req_distinguished_name
 [req_distinguished_name]
 [ v3_req ]
 basicConstraints = CA:FALSE
-keyUsage = nonRepudiation, digitalSignature, keyEncipherment
```
Comment (caesarxuchao) [inline plugin/pkg/admission/webhook/gencerts.sh]: Could you explain why we need to remove this line?

**Comment by caesarxuchao** [comment]
Removing the SAN from CSR looks reasonable. I happened to test a similar example in my local set up recently and it worked, so that part lgtm. Thanks.

File: plugin/pkg/admission/webhook/gencerts.sh (hunk 1 above)
Comment (dqminh) [reply plugin/pkg/admission/webhook/gencerts.sh]: no, its a typo! i should not remove it.

**Comment by dqminh** [comment]
@caesarxuchao looks like the test flaked, do you mind kick the test again ?

**Comment by dims** [comment]
already fixed! https://github.com/kubernetes/kubernetes/commit/229cbf4b6d9ab10ac509cb3b53aa1b169f58afd9

**Comment by dims** [comment]
@dqminh let's close this one

**Pull request closed without merging**